model_compression_toolkit/core/pytorch/pytorch_implementation.py

```diff
@@ -474,3 +516,66 @@ def get_node_mac_operations(self,
             return kernel_shape[0] * kernel_shape[1]
         else:
             return 0
+
+    def apply_second_moment_correction(self, quantized_model, core_config, representative_data_gen, graph):
+        """
```

Comment:
Maybe it's worth considering writing this method in a dedicated file and importing it (Relevant for Keras as well)?